src/font_loader.h

```diff
@@ -35,7 +36,13 @@ class font_loader
                 std::ifstream stream( path.c_str(), std::ifstream::binary );
                 JsonIn json( stream );
                 JsonObject config = json.get_object();
-                config.read( "typeface", typeface );
+                if( config.has_string( "typeface" ) ) {
+                    std::string tmp;
+                    config.read( "typeface", tmp );
+                    typeface.emplace_back( tmp );
```

Comment:
```suggestion
                    typeface.emplace_back( config.get_string( "typeface" ) );
```
No need for a variable.